### DISCUSSION TITLE
Walker waypoints and highlighted blue range not showing up

### DISCUSSION THREAD

**Discussion by prithvikukillaya** [Keriew/augustus#900] [Q&A]
Does anyone know which setting toggle enables you to see the walk waypoints and ranges when placing buildings? I tried the setting toggle I thought it would be (the one related to range) but it doesn't seem to make a difference to my game. I can see it. I'm wondering if there are any other settings that would cause a conflict and prevent this from being displayed.

**Comment by crudelios** [comment]
Are you using the unstable version? The latest stable v3.2.0 doesn't have it yet.
